src/OpenTelemetry.Exporter.OpenTelemetryProtocol/OtlpMetricExporterExtensions.cs

```diff
@@ -113,8 +127,7 @@ public static MeterProviderBuilder AddOtlpExporter(this MeterProviderBuilder bui
 
             builder.ConfigureServices(services =>
             {
-                services.RegisterOptionsFactory(configuration
-                    => new OtlpExporterOptions(configuration, defaultBatchOptions: null));
+                OtlpTraceExporterHelperExtensions.RegisterOtlpExporterOptionsFactory(services);
```

Comment:
nit: Kinda weird to have this method in `OtlpTraceExporterHelperExtensions`. I don't feel strongly this needs to change right now.

Reply:
I moved the internal helper extension onto `OtlpExporterOptions` that seemed like a _slightly_ better spot for it? 🤣